### ISSUE TITLE
missing folder option

### ISSUE THREAD

**Issue by joaociocca** [HeyParkerJ/batch_megadl#1]
Considering [Marco's original solution](https://github.com/megous/megatools/issues/157#issuecomment-615835778)

    it works for folders links too, just replace `folder/` with `#F!` and `#` with `!`

A folder template link would look like this: `https://mega.nz/folder/<some_id>#<some_other_id>` and to work with megadl would have to be changed into this: `https://mega.co.nz/#F!<some_id>!<some_other_id>`

**Comment by alfredonodo** [comment]
Hi,
it seems that does not work anymore starting from version 1.11.0. Any hint?

**Comment by HeyParkerJ** [comment]
Hey @joaociocca thanks for the help figuring that out! I didn't notice your issue until now. Unfortunately my Linux box is on the fritz at the moment and I won't be able to get to this until the new year, but I'll approve any PR that implements handling folders if anyone gets to it before me. Alternatively I can implement this within the next month or so.

**Comment by alfredonodo** [comment]
> Hi, it seems that does not work anymore starting from version 1.11.0. Any hint?

I found that the version 1.11.0 is compatible with the new format `https://mega.nz/folder/<some_id>#<some_other_id>` instead of the previous one `https://mega.co.nz/#F!<some_id>!<some_other_id>`

**Comment by joaociocca** [comment]
> > Hi, it seems that does not work anymore starting from version 1.11.0. Any hint?
> 
> I found that the version 1.11.0 is compatible with the new format `https://mega.nz/folder/<some_id>#<some_other_id>` instead of the previous one `https://mega.co.nz/#F!<some_id>!<some_other_id>`

it may be, but 1.11 is still unavailable on `focal` repo.
```bash
$ sudo apt search megatools
Sorting... Done
Full Text Search... Done
megatools/focal,now 1.10.2-1 amd64 [installed]
  Command-line client for the Mega cloud storage service
```

Also, megous moved away from github to a self-hosted, at https://megous.com/git/megatools - and the latest version there is still 1.10.2. Looking around there, it seems 1.10.3 was developed last year to account for this exact problem, but never left the 1.10.x branch https://megous.com/git/megatools/log/?h=1.10.x

**Comment by alfredonodo** [comment]
> > > Hi, it seems that does not work anymore starting from version 1.11.0. Any hint?
> > 
> > 
> > I found that the version 1.11.0 is compatible with the new format `https://mega.nz/folder/<some_id>#<some_other_id>` instead of the previous one `https://mega.co.nz/#F!<some_id>!<some_other_id>`
> 
> it may be, but 1.11 is still unavailable on `focal` repo.
> 
> ```shell
> $ sudo apt search megatools
> Sorting... Done
> Full Text Search... Done
> megatools/focal,now 1.10.2-1 amd64 [installed]
>   Command-line client for the Mega cloud storage service
> ```
> 
> Also, megous moved away from github to a self-hosted, at https://megous.com/git/megatools - and the latest version there is still 1.10.2. Looking around there, it seems 1.10.3 was developed last year to account for this exact problem, but never left the 1.10.x branch https://megous.com/git/megatools/log/?h=1.10.x

You are right, I have 1.10.2 on ubuntu 20.04.x while I have 1.11.0 on windows 10 via [chocolatey package manager.](https://community.chocolatey.org/packages/megatools#versionhistory)
